custom/templates/DefaultRevamp/footer.tpl

```diff
@@ -12,7 +12,19 @@
                     {if $PAGE_LOAD_TIME}
                     <span class="item" id="page_load"></span>
                     {/if}
-                    <a class="item" href="javascript:" onclick="toggleDarkLightMode()">{$TOGGLE_DARK_MODE_TEXT}</a>
+                    <input type="checkbox" class="darkmode-toggle" id="darkmode-toggle" onclick="toggleDarkLightMode()">
+                    <label for="darkmode-toggle" class="darkmode-toggle-label">
+                        <i class="fas fa-moon"></i>
+                        <i class="fas fa-sun"></i>
+                        <div class="darkmode-ball"></div>
+                    </label>
+                    <script type="text/javascript">
+                        if (document.body.classList.contains('dark')) {
+                            document.getElementById("darkmode-toggle").checked = true;
+                        } else {
+                            document.getElementById("darkmode-toggle").checked = false;
+                        }
+                    </script>
```

Comment:
Should be at the bottom

Reply:
I'd like to hear if other contributors agree as this script isn't meant to be global, so to speak.

Reply:
I'm happy for this to be inline like this 👍